src/firebase.ts

```diff
@@ -57,10 +59,7 @@ export function useEmulatedFirebaseApp(name: string, config: any) {
   const projectId = useProjectId();
 
   const app = useMemo(() => {
-    const app = firebase.initializeApp(
-      { ...config, projectId },
-      `${name} component::${Math.random()}`
-    );
+    const app = firebase.initializeApp({ ...config, projectId });
```

Comment:
Please use a unique app ID or this may throw upon emulator restart (`config` and `projectId` may change / flicker). I'd suggest reverting this change.